lib/template.html

```diff
@@ -219,28 +219,60 @@
     "headline": {{ seo_page_title | jsonify }},
 {% endif %}
 
+{% if seo_author %}
+    "author": {
+      "@type": "Person",
+      "name": {{ seo_author | jsonify }}
+    },
+{% endif %}
+
 {% if seo_page_image %}
-    "image": {{ seo_page_image | jsonify }},
+    {% if page.image.height && page.image.width %}
+      "image": {
+          "@type": "ImageObject",
+          "url": {{ seo_page_image | jsonify }},
+          "height": {{ page.image.height | jsonify }},
+          "width": {{ page.image.width | jsonify }}
+      },
+    {% else %}
+        "image": {{ seo_page_image | jsonify }},
+    {% endif %}
 {% endif %}
 
 {% if page.date %}
     "datePublished": {{ page.date | date_to_xmlschema | jsonify }},
 {% endif %}
 
+{% if page.date_modified %}
+    "dateModified": {{ page.date_modified | date_to_xmlschema | jsonify }},
+{% elsif page.date %}
+    "dateModified": {{ page.date | date_to_xmlschema | jsonify }},
```

Comment:
Why include the date modified if it's really the publish date? Is there value in doing this?